src/client/sandbox/event/message.js

```diff
@@ -207,8 +207,7 @@ export default class MessageSandbox extends SandboxBase {
                     // trying to call the 'isCrossDomainWindows' function, so we have to wrap it in 'try/catch'.
                     try {
                         targetWindow[this.RECEIVE_MSG_FN]({
-                            // NOTE: Cloning a message to prevent this modification.
```

Comment:
I removed the test which checks that message was cloned.
https://github.com/DevExpress/testcafe-hammerhead/pull/1249/files#diff-366a593bdf9d1d4547600cef78eedf11L175